app/src/main/java/net/gsantner/markor/ui/FilesystemViewerCreator.java

```diff
@@ -56,10 +54,10 @@
 
         opts.accentColor = R.color.accent;
         opts.primaryColor = R.color.primary;
-        opts.primaryTextColor = darkTheme ? R.color.dark__primary_text : R.color.light__primary_text;
-        opts.secondaryTextColor = darkTheme ? R.color.dark__secondary_text : R.color.light__secondary_text;
-        opts.backgroundColor = darkTheme ? R.color.dark__background : R.color.light__background;
-        opts.titleTextColor = titleLight ? R.color.dark__primary_text : R.color.light__primary_text;
+        opts.primaryTextColor = R.color.primary_text;
+        opts.secondaryTextColor = R.color.secondary_text;
+        opts.backgroundColor = R.color.background;
+        opts.titleTextColor = R.color.primary_text;
```

Comment:
Do you want to remove these options and just move them into the layouts?

Reply:
The named colors from overall palette could be moved to theme yes.

![grafik](https://user-images.githubusercontent.com/6735650/147790243-563d07c5-d520-43bc-a5aa-142ac41a506a.png)

Preferably in a second PR, to not overlook something here